integration/integrate-with-email.adoc

```diff
@@ -7,17 +7,43 @@ include::modules/common-attributes.adoc[]
 
-You can use email as a notification method by forwarding alerts from {product-title-short} to a standard email provider. To forward alerts from the {product-title-short} platform to an email address, you can use the `Default Recipient` field to send email to a standard and centralized team, or use deployment annotations to specify an audience for notifications.
+To forward alerts from {product-title-short} and the {product-title-managed-short} to an email address, you can use the `Default recipient` field to send emails to a standard and centralized team or use deployment annotations to specify an audience for notifications.
 
-With annotation keys, you can define an audience to notify about policy violations that are associated with a deployment or namespace. If the deployment has an annotation, the annotation's value overrides the default value. If the namespace has an annotation, the namespace's value overrides the default value.
+With annotation keys, you can define an audience to notify about policy violations associated with a deployment or namespace.
+The following rules govern how {product-title-short} determines the recipient of an email notification:
 
-* If a deployment has an annotation key and a defined audience, an email is sent to the audience who is defined by the key.
-* If a deployment does not have an annotation key, the namespace is checked for an annotation key and an email is sent to the defined audience.
-* If no annotation keys exist, an email is sent to the default recipient that is defined in the integration.
+* If the deployment has an annotation key, the annotation's value overrides the default value.
+* If the namespace has an annotation key, the namespace's value overrides the default value.
+* If a deployment has an annotation key and a defined audience, {product-title-short} sends an email to the audience specified in the key.
+* If a deployment does not have an annotation key, {product-title-short} checks the namespace for an annotation key and sends an email to the specified audience.
+* If no annotation keys exist, {product-title-short} sends an email to the default recipient.
 
-:toolname: email
-include::modules/configure-acs-kube-security-platform.adoc[leveloffset=+1]
+== Integrating with email on {product-title-short}
 
-include::modules/configure-policy-notifications.adoc[leveloffset=+1]
+You can use email as a notification method by forwarding alerts from {product-title-short}.
+
+include::modules/configure-acs-email.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../operating/manage-vulnerabilities/vulnerability-management.adoc#vulnerability-management20-configure-report-delivery-destinations-schedule_acs-operating-manage-vulnerabilities[Configuring delivery destinations and scheduling]
+
+:toolname: Email
+:context: rhacs
+include::modules/configure-policy-notifications.adoc[leveloffset=+2]
+
+== Integrating with email on {product-title-managed-short}
+
```

Comment:
@gaurav-nelson I just realized that the current doc can leave the impression that you can't use the "old" email integration when in the cloud, which is not true.

You can use both types of integration. The "old" email integration where you can "bring your own SMTP service" and the new acscsEmail integration where emails get send through a managed SMTP service.

I think it would be good to make that more clear in the doc.